### ISSUE TITLE
TLS 1.2 support available?

### ISSUE THREAD

**Issue by WiktorBAbove** [espressif/esp-wolfssl#3]
Hi,
Is there a list describing which methods that are supported or not supported in the static lib?
We are wondering mainly if TLS 1.2 is supported.
We try to do a wolfTLSv1_2_client_method() but according to our debug session it does a wolfTLSv1_3_client_method().
But we see that README.md says that:
"The binary library does not include wolfSSL features for TLS 1.3, FIPS or DTLS 1.2."
BR /W

**Comment by AdityaHPatwardhan** [comment]
Hello @WiktorBAbove, Thanks for the issue, and Yes TLS1.2 is fully supported in wolfSSL, as far as your debug output I am assuming you are using code from the patch I provided. Well in that patch we have used `wolfSSLv23_method` for client/server `ctx` initialisation, in that place if you just use `wolfTLSv1_2_client_method()`, like we have already used in the master branch ( ` before patch`),  all tls1.2 functions will be used for session. and thank you for notifying about the `README.md` , actually we yet have to test overall working of `TLS1.3` of wolfSSL so we did not change that line which you mentioned.
